### ISSUE TITLE
Accessibility Issues with Text Style Settings pop-Up

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32171]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

In the Text Style Settings Pop-up navigation gets stuck in spin boxes making some items difficult to navigate to.  The layout and how to navigate this area is very confusing and a number of items are not labelled individually.

STEPS:
1.  Navigate to  text element such as the tempo mark.
2. Press F2 and then Tab to move on to the Text Style Settings area.
3. Press right arrow twice and you land on the font size area. Note that if you arrow in any direction now you only change the value.  It is not obvious that Esc will close this box and allow navigation to resume as usually Esc would return you to the score closing the edit area.  The user will not want to press enter because they can't hear what the current setting is (see below).  It would be more usual for these boxes to only change increment with up/down arrows and other navigation continue with left/right arrows.
4. If you continue to arrow down you come to a series of buttons that announce as "Vertical Alignment buttons" and "Text Style Buttons".  I'm guessing the Text style ones might be bold, italic, underline, etc?  All of these say the same thing except one announces as checked and the rest not checked.  There is no way to differentiate them.
5. Further arrowing seems to get stuck again in the line spacing box.
6. Exploring with down arrow and then with right arrow seems to come across different items.  A blind user can not understand the order or layout of this panel to know where to move to find things, it makes it feel like a maze.  The usual thing would be to Tab to each new item, but as Tab jumps to the next panel it would be better if this consistently moved through all items in order with either left/right or up/down arrows and did not allow movement with the other.
7. Arrow on to the font size area and notice it announces "font size spin button" but does not announce the current setting.  Because arrows now all move in all directions it is not possible to know what the current setting is as you can't easily move on/off to hear it announce.  It should announce the current value when you enter it.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
Just noting that in other places in MuseScore Studio 4 it seems usual for these spin boxes not to open automatically when arrowed over them, but to use left/right arrows to navigate to each item and press enter to open them.  For example the speed button in the play back settings.  It is then logical if you have pressed enter to open something to close with Esc.  We really need some consistency of behaviour across the programme.
